dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/cookies/CookieUtils.java

```diff
@@ -578,7 +600,12 @@ private static void setAttributes(HttpCookie cookie, StringBuilder buffer) {
             String key, value;
             for (Entry<String, String> entry : cookieAttrs.entrySet()) {
                 key = entry.getKey();
-                if (!(key.equals("samesite") || key.equals("port") || key.equals("commenturl"))) {
+                // add partitioned if not in beta.
+                // skip if in beta since it's added above already
+                if((key.equals("partitioned") && ProductInfo.getBetaEdition())){
+                    continue;
+                }
+                if (!(key.equals("samesite") || key.equals("port") || key.equals("commenturl")) ) {
```

Comment:
This part is tested by tested by Servlet60CookieSetAttributeTest's  `test_setAttributePartitioned` -- we expected `partitioned=` to be set if not in beta; otherwise, it's just `; Partitioned`

Reply:
Not sure I'm following here, are we making a change that would work outside of Beta in a GA release?